### DISCUSSION TITLE
Proposal for a "Releases" Category

### DISCUSSION THREAD

**Discussion by bjlittle** [SciTools/iris#4069] [Releases]
Each repository can have up to [10 Discussions Categories](https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-categories-for-discussions-in-your-repository#about-categories-for-discussions).

I propose that we start a **Releases** category as an open ended discussion (rather than a question/answer category) dedicated to Iris releases.

We could have a thread per release e.g., **v3.0.x**, **v3.1.x** etc where each thread contains any and all information relevant to that release feature branch e.g.,

- The proposed/scheduled date for a forthcoming release
- The release manager
- Freeze and chill dates
- Links referencing the release milestone, project, tags and releases
- Details on packaging progress of the release to conda-forge and PyPI
- Details on pending patch releases such as content, due dates etc
- General release specific announcements

Essentially, a one stop shop for the community to garner **dynamic** information regarding the status and progress of a release. My hope is that it would allow the community to better track the progress of a release, but more importantly touch base with developers and the release manager.

Obviously, this would complement our existing RTD documentation (release timelines/roadmaps, nudge, wink @tkknight) for both stable and latest. However, having an **active** and **curated** discussion thread would perhaps help fill the void in release communication for the community, as they only ever tend to know about a release once it's actually happened.

Ideally, I'd love to make the release process way more transparent and accessible to the community, as an end-to-end process. Having an active Releases discussion category may also help us as core developers be more responsive and aware of community priorities and requirements regarding an active and/or scheduled release. 

Anyways, have your say... what do you think?

**Comment by bjlittle** [comment]
A few votes for the proposal from @tkknight and @trexfeathers, which is enough for me... let's roll with that and see how it goes :smile: 

Thanks guys :+1:

**Comment by pp-mo** [comment]
Archiving "answered" Q+As

**Comment by rcomer** [comment]
Discussions can now be closed 👀
https://github.blog/changelog/2023-03-14-adding-the-ability-to-close-github-discussions/
